irohad/validation/impl/stateful_validator_impl.cpp

```diff
@@ -34,18 +36,21 @@ namespace iroha {
         ametsuchi::TemporaryWsv &temporaryWsv) {
       log_->info("transactions in proposal: {}", proposal.transactions.size());
       auto checking_transaction = [this](auto &tx, auto &queries) {
-        return (queries.getAccount(tx.creator_account_id) |
+        auto transaction =
```

Comment:
There is not required to convert the transaction to old-model. Use conversion on-demand.